src/components/ViewPost/ViewPost.jsx

```diff
@@ -0,0 +1,165 @@
+import React from 'react'; 
+import useStyles from "./style";
+import { ThemeProvider } from '@material-ui/styles';
+import {theme} from "./style";
+import {Container,
+        Grid, 
+        Typography, 
+        CardMedia, 
+        Divider, 
+        ListItem,
+        Paper,
+        } from '@material-ui/core';
+
+
+function ViewPost() {
+  const classes = useStyles();
+
+  return (
+    <>
+        <Container className={classes.container}>
+            <Paper className={classes.paper} >
+                <ThemeProvider theme={theme}>
+                    <Typography variant="h4" className={classes.headerTypo}>
+                        Item Lost Information:
+                    </Typography>
+                    <Grid container>
+                        <Grid xs={16} sm={8} md={3}>
+                            <CardMedia
+                                className={classes.media}
+                                image="https://miro.medium.com/max/1200/1*mk1-6aYaf_Bes1E3Imhc0A.jpeg"
+                                title=""
+                            />
+                        </Grid>
+                        <Grid item xs={12} sm={6} md={8}>
+                            <ListItem>
+                                <Container maxWidth="lg" className={classes.container2}>
+                                    <Typography variant="body1" color="textSecondary">
+                                        Item Name:
+                                    </Typography>
+                                    <Typography variant="body1">
```

Comment:
Not sure about this butt `body1` is not a good name, if possible make it meaningful!

Reply:
this is a variant from material UI.